public/lang/locale-de-main.json

```diff
@@ -239,5 +239,10 @@
     "Bahnhofpläne": "Bahnhofpläne",
     "Printprodukte": "Printprodukte",
     "Plakat": "Plakat",
-    "Format A4": "Format A4"
+    "Format A4": "Format A4",
+
```

Comment:
Why there is spaces ?

Reply:
IT was for categorization of translations, but we will organize better this files later.